### ISSUE TITLE
Einspeise Zählerstand "unknown"

### ISSUE THREAD

**Issue by compumark** [DarwinsBuddy/WienerNetzeSmartmeter#146]
Hi,

I have connected my HA with this plugin via HACS to the Wiener Netze Smartmeter. 
The meter with the "Bezug" I am correctly receiving daily. 
I also have a second meterpoint for the produced PV. This I can see in the smartmeter website from Wiener Netzte and the integrations reads is correctly, but the data is always "unknown".
The attributes in the integration are correct shown. 

And a second question: Is it possible to get also the 181 and 182 values instead only the 180?
thanks a lot for any help.
Mark

**Comment by Kuschla** [comment]
Hi, 
I've added the integration a 2nd time - in the 1st version, I have the same like you describe. In the 2nd, it's vice-versa "Bezug" is unknown, but the feed-in point shows statistical data. You can then add the one (or other) Smart Meter point to the energy Dashboard.

**Comment by compumark** [comment]
Hi,
I tried that yesterday and today it looked like this again:

![image](https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/assets/75847830/dc6aa4a7-d111-4957-8957-344a672d1d43)

The second one has again 2 entities:
![image](https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/assets/75847830/1b8591c1-3205-4878-a15f-9f52e7236324)

the other one, 0 entities. 
![image](https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/assets/75847830/54a0bc82-c081-43d0-bfec-b6e2a1df3bb0)

I will try to delete it now and create 2 new integrations.
let see, if that works

**Comment by compumark** [comment]
After some days of checking, I can say that I now get values for the Einspeisung, but those seems to be daily values and noch the counter of the smartmeter like the Bezug values... Very strange

**Comment by compumark** [comment]
Hi @Kuschla 
I monitored it now a couple of days and come to the conclusion:

The Zählerpunkt marked as favorite in the smartmeter portal 
![image](https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/assets/75847830/b48c6b41-6f5a-46ff-8015-e7b4b1add8ac)

gets the correct Zählerstand transferred - if they are available in the portal.
The other - non favorite Zählerpunkt - seems to get the used kWh from the day, not the Zählerstand. 
![image](https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/assets/75847830/d6c64526-73d4-4a56-808d-0e3736f0b5e2)

In the screenshot above I just swapped the favorite to the Bezugszähler in the smartmeter portal and the next refresh reads the total used kWh correctly, but the Einspeisung swaps back to the kWh delivered to the Zählerstand.
